paths_cli/wizard/errors.py

```diff
@@ -1,23 +1,42 @@
 class ImpossibleError(Exception):
+    """Error to throw for sections that should be unreachable code"""
     def __init__(self, msg=None):
         if msg is None:
             msg = "Something went really wrong. You should never see this."
         super().__init__(msg)
 
+
 class RestartObjectException(BaseException):
+    """Exception to indicate the restart of an object.
+
+    Raised when the user issues a command to cause an object restart.
+    """
     pass
 
+
 def not_installed(wizard, package, obj_type):
+    """Behavior when an integration is not installed.
+
+    In actual practice, this calling code should ensure this doesn't get
+    used. However, we keep it around as a defensive practice.
+
+    Parameters
+    ----------
+    package : str
+        name of the missing package
+    obj_type : str
+        name of the object category that would have been created
+    """
     retry = wizard.ask(f"Hey, it looks like you don't have {package} "
                        "installed. Do you want to try a different "
                        f"{obj_type}, or do you want to quit?",
                        options=["[R]etry", "[Q]uit"])
     if retry == 'r':
         raise RestartObjectException()
-    elif retry == 'q':
+    if retry == 'q':
+        # TODO: maybe raise QuitWizard instead?
```

Comment:
Does this handle `Q` as well? (as that is mentioned in the `options`) (same question about `R` for line `34`)

Reply:
Not yet. A future improvement will be to make things case insensitive (and also a little more flexible in matching.) Currently , when given an option like:

```
🧙 What will you use for the underlying engine?

      1. OpenMM
      2. Load existing from OPS file

🧙 Please select an option: 
```

you can either type `1` to get OpenMM, or you can type `OpenMM` (but you must exactly match case and text). I'd like to make this match based on case-insensitive partial matches. For example, typing `LOAD` for that prompt should be unambiguous enough that we select option 2. Rather than handle that in a piece-by-piece way, (e.g., by supporting `!Q` now), my current preference is to try to get it all in one broader change.

Reply:
(Also, some inputs, like object names, will always be case sensitive)

Reply:
Ah, for this case do you mind aligning it with the `options` on line 33, so either making this  `Q` or make the option lowercase "[q]uit" (and the same for the "R" or "[r]etry")?

Reply:
Done in 3fb1571 (Although I'm not sure whether those options there are actually used... these are the sorts of things that I want to clean up in some future refactoring.)